### PR TITLE
Likes feature

### PR DESCRIPTION
**Pull request by ghassanmas**
Adding the like feature, now you can like the post. 
Changes are already deployed http://facn.herokuapp.com/

### CODE REVIEW

**Review by SamiSha99 — CHANGES_REQUESTED** [review]
I would like to see a way to block self liking your own threads, obviously because there is no point in it. https://i.imgur.com/tlx8Yhv.png

**Comment by ghassanmas** [comment]
This is a subjective opinion, people may like their posts for different reasons:
- When they post something they really like compared to other posts.  
- To enhance the possibility that they get more likes. 
- It could be sort of sarcasm.

Regardless of my opinion, I would find it very strict to band the users from liking their own posts.

**Comment by Amirk390** [comment]
Reviewing it now :smile:

File: public/css/style.css [hunk 1]
```diff
@@ -7,6 +7,96 @@ body {
...
+  border: 16px solid #f3f3f3;
+  border-radius: 50%;
+  border-top: 16px solid #3498db;
+  width: 120px;
+  height: 120px;
+  -webkit-animation: spin 2s linear infinite;
+  animation: spin 2s linear infinite;
+  display: none;
+}
+.likes {
```
Comment (Amirk390) [inline public/css/style.css:32]: The likes are good idea!
can you just fix the alignments of the likes? 

add maybe a div for the likes and the h4 number of likes so they will be aligned `align-items: center;`

**Review by Amirk390 — CHANGES_REQUESTED** [review]
Good job :clap: 

just 1 minor change :smile:

File: public/css/style.css (hunk 1 above)
Comment (ghassanmas) [reply public/css/style.css:32]: yes there was styling issues!, now fixed ! :tada: :tada:

**Review by Amirk390 — APPROVED** [review]
looks good now :clap:

**Comment by Amirk390** [comment]
@SamiSha99 

> I would like to see a way to block self liking your own threads, obviously because there is no point in it. https://i.imgur.com/tlx8Yhv.png

No reason to block it,
its the same in every social community

**Review by SamiSha99 — APPROVED** [review]
I suppose since every social media has it, I can give it a pass.

**Comment by sophiavocado** [comment]
aweeeeesommeeeeeee job!

**Pull request merged**